source/install/virtual-machine.rst

```diff
@@ -17,21 +17,113 @@ The core distribution builds contain the following plugins:
+.. note:: At this stage we're unsure of what types of virtual machine builds we'll continue to release in the future. We expect that for the first half of 2017 we'll provide the three builds included here, but we may ultimately phase out the AWS and VirtualBox builds. This is because our release of these builds restricts users to a specific operating system/cloud platform/etc, so we feel that we may better serve our user community by providing cross-platform builds (e.g., Docker containers) and detailed instructions for how to work with these.
 
 VirtualBox
 ----------
 
-1. Set up VirtualBox on your computer (see https://VirtualBox.org for details).
-2. Download the QIIME 2 Core VirtualBox Image here: https://data.qiime2.org/distro/core/qiime206.zip.
-3. Unzip the ``qiime206.zip`` file.
-4. Launch VirtualBox, and create a new machine (click "New").
-5. A new window will show up. Click "Next".
-6. In this screen type *QIIME 2 Core 2.0.6* as the name for the virtual machine then select Linux as the Operating System, and Ubuntu (64 bit) as the version. Click "Next".
-7. Select the amount of RAM (memory). You will need at least 2048 MB, but the best option is based on your machine. After selecting the amount of RAM. Click "Next".
-8. Select "Use existing hard drive", and click the folder icon next to the selector (it has a green up arrow). In the new window click "Add", and locate the virtual hard drive that was unzipped in step 3. Click "Select" and then click "Next".
-9. In the new window, click "Finish".
-10. Launch the virtual machine. When prompted, select the user ``qiime2``, and enter the password ``qiime2``.
+1. Install VirtualBox on your computer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please see https://VirtualBox.org for details on how to install VirtualBox on your computer. Please note, some computers require activating a BIOS setting to enable virtualization --- please consult your hardware documentation to determine if you need to do this, and how.
+
+
+2. Download the QIIME 2 Core VirtualBox Image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please note, this installation document uses some placeholders throughout the following steps. ``X.Y.Z`` represents the version of QIIME 2 (e.g. ``2017.2.0``), and ``build_number`` represents the build identifier for the VirtualBox image (an internal reference used by the QIIME 2 Developement Team). If you select the "Browser" option below, the file will automatically rename to the version and build number necessary.
+
+.. download::
+   :no-exec:
+   :url: https://data.qiime2.org/distro/core/latest
+   :saveas: qiimeX_Y_Z-build_number.zip
+
+3. Unzip the file
+~~~~~~~~~~~~~~~~~
+
+Unzip the ``qiimeX_Y_Z-build_number.zip`` file and open the directory that you chose to unzip to.
+
+4. Launch VirtualBox
+~~~~~~~~~~~~~~~~~~~~
+
+Double-click the file named ``QIIME 2 Core - X.Y.Z (build_number).ovf``.
```

Comment:
Can you add a `.. note::` directive somewhere in this section describing what the .vmdk file is and why you might use it?